src/main/java/by/custom/utilcalculator/domain/UserProgressStorage.java

```diff
@@ -1,27 +1,52 @@
 package by.custom.utilcalculator.domain;
 
+import org.telegram.telegrambots.meta.api.objects.Message;
+
+import java.io.*;
 import java.util.HashMap;
 
 public class UserProgressStorage {
```

Comment:
Please, create interface for UserProgressStorage. Could be done in future. Lets discuss it during call.
```suggestion
public class UserProgressStorage {
```

Reply:
Realized. Looks great